docs/book/v3/file.md

```diff
@@ -140,13 +137,15 @@ Configure different options for several possible source files:
 ```php
 $filter = new \Laminas\Filter\File\Rename([
     [
-        'source'    => 'fileA.txt'
-        'target'    => '/dest1/newfileA.txt',
+        'match'    => 'fileA.txt'
+        'target_directory' => '/dest1/',
+        'rename_to'    => 'newfileA.txt',
         'overwrite' => true,
     ],
     [
-        'source'    => 'fileB.txt'
-        'target'    => '/dest2/newfileB.txt',
+        'match'    => 'fileB.txt'
+        'target_directory' => '/dest2/',
+        'rename_to'    => 'newfileB.txt',
```

Comment:
```suggestion
        'match'    => '*.txt'
        'target_directory' => '/text-files/',
        'rename_to'    => 'newfileB.txt',
```

Changes config to show the use-case of moving different types of file to different target directories